main.go

```diff
@@ -66,7 +69,10 @@ func main() {
 		outfile = hash + ".png"
 	}
 
-	fmt.Printf("Creating size %v avatar for hash %v, writing into %v\n", size, hash, outfile)
+	if !writeStdout {
+		fmt.Printf("Creating size %v avatar for hash %v, writing into %v\n", size, hash, outfile)
+	}
```

Comment:
I think we should just send this to stderr instead of disabling it. Not sure why I sent this to stdout and the rest to stderr -- it should all go to stderr.